packages/office-ui-fabric-react/src/components/Sticky/Sticky.tsx

```diff
@@ -161,13 +172,23 @@ export class Sticky extends BaseComponent<IStickyProps, IStickyState> {
   }
 
   public resetSticky(): void {
-    if (this.nonStickyContent && this.root) {
-      this.root.appendChild(this.nonStickyContent);
+    if (this.nonStickyContent && this.placeholder) {
+      this.placeholder.appendChild(this.nonStickyContent);
     }
   }
 
   public setDistanceFromTop(container: HTMLDivElement): void {
-    this.distanceFromTop = this._getNonStickyDistanceFromTop(container);
+    this._setDistanceFromTop(this._getNonStickyDistanceFromTop(container));
+  }
+
+  private _setDistanceFromTop(distance: number): void {
+    const { scrollablePane } = this.context;
+
+    if (this.distanceFromTop !== distance && scrollablePane) {
+      this.distanceFromTop = distance;
+      scrollablePane.sortSticky(this, true);
+      this.forceUpdate();
```

Comment:
We can't have `forceUpdate` called continuously on scrolling; that would drag down performance immensely.